apis/v1alpha1/convert.go

```diff
@@ -112,7 +112,8 @@ func tov1beta1(in OpenTelemetryCollector) (v1beta1.OpenTelemetryCollector, error
 				AdditionalContainers:          copy.Spec.AdditionalContainers,
 			},
 			StatefulSetCommonFields: v1beta1.StatefulSetCommonFields{
-				VolumeClaimTemplates: copy.Spec.VolumeClaimTemplates,
+				VolumeClaimTemplates:                 copy.Spec.VolumeClaimTemplates,
```

Comment:
You don't need to add this to v1alpha1 or the conversion webhook. This version is deprecated and we try not to make changes to it.